SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -230,6 +233,9 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         
         performEndCallAction(uuid: call.uuid)
         callDisconnected()
+        
+        // Restore to audio enabled
+        TwilioVoice.isAudioEnabled = true
```

Comment:
I don't think this is necessary. Can we defer to `provider:didDeactivateAudioSession` instead?